src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/TypeWitness.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Text.Json.Reflection
+{
+    /// <summary>
+    /// Encapsulates a generic type parameter that can be unpacked on-demand using a generic visitor,
+    /// encoding an existential type. Can use reflection to instantiate type parameters that are not known at compile time.
+    /// Uses concepts taken from https://www.microsoft.com/en-us/research/publication/generalized-algebraic-data-types-and-object-oriented-programming/
```

Comment:
IMO this link is unnecessary; if kept remove "/en-us"

Reply:
Updated, thanks.